src/query/api/v1/httpd/handler.go

```diff
@@ -55,10 +54,14 @@ import (
 )
 
 const (
-	healthURL        = "/health"
-	routesURL        = "/routes"
-	engineHeaderName = "M3-Engine"
-	engineURLParam   = "engine"
+	healthURL = "/health"
+	routesURL = "/routes"
+	// EngineHeaderName defines header name which is used to switch between
+	// prometheus and m3query engines.
+	EngineHeaderName = "M3-Engine"
+	// EngineURLParam defines query url parameter which is used to switch between
+	// prometheus and m3query engines.
+	EngineURLParam = "engine"
```

Comment:
Do you know if there "engine" param would be used? In general, it's on master already (and in docs) so I think it would be nice to do not change it until we are sure that it's can't be used.

Reply:
It's not used at the moment, just future-proofing considerations ;) 

Probably being overly paranoid, all good to leave it :+1: